package.json

```diff
@@ -44,7 +44,7 @@
     "css-loader": "^0.28.9",
     "enzyme": "^3.3.0",
     "eslint-config-edx": "^4.0.3",
-    "extract-text-webpack-plugin": "^3.0.2",
+    "extract-text-webpack-plugin": "next",
```

Comment:
Unfortunately, extract-text-webpack-plugin's webpack 4 support is still in beta right now.

Reply:
Can you add an issue to bump this when it's not in beta?

Reply:
Just so we can link things for historical purposes... 

#12 (thanks @thallada !)